src/js/components/Ready/VoterPlanModal.jsx

```diff
@@ -165,8 +165,9 @@ class VoterPlanModal extends Component {
     const googleCivicElectionId = VoterStore.electionId();
     const savedVoterPlan = ReadyStore.getVoterPlanForVoterByElectionId(googleCivicElectionId);
     this.setVoterPlanSavedStates(savedVoterPlan);
+    const voterPlansForVoterRetrieved = ReadyStore.getVoterPlansForVoterRetrieved();
     let savedVoterPlanFound = false;
-    if (savedVoterPlan.google_civic_election_id === undefined) {
+    if (!voterPlansForVoterRetrieved) {
       ReadyActions.voterPlansForVoterRetrieve();
     } else {
       savedVoterPlanFound = true;
```

Comment:
Hi @mlevoy I think we still wan an `else if (savedVoterPlan.google_civic_election_id)` test before marking "savedVoterPlanFound = true"